docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`5978` rotki will now properly decode the swaps done via 0x protocol.
```

Comment:
We discussed in discord that the only thing remaining is the uniswap trades in other cahins. Since uniswap is not done yet and the 0x feature is not fully supported we will add it in other PR the changelog entry once fully integrated. But leaving it here in case there are more comments in the PR